### PR TITLE
Add Tautulli chart

### PR DESCRIPTION
**Pull request by onedr0p**
Please review and let me know if you need any changes.

Thanks!

Closes https://github.com/billimek/billimek-charts/issues/90

### CODE REVIEW

File: tautulli/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# NZBGet Usenet client
```
Comment (billimek) [inline tautulli/README.md]: This heading should say `Tautulli` instead of `NZBGet`

File: tautulli/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+|----------------------------|-------------------------------------|---------------------------------------------------------|
+| `image.repository`         | Image repository | `linuxserver/tautulli` |
+| `image.tag`                | Image tag. Possible values listed [here](https://hub.docker.com/r/linuxserver/tautulli/tags/).| `v2.1.39-ls32`|
+| `image.pullPolicy`         | Image pull policy | `IfNotPresent` |
+| `strategyType`             | Specifies the strategy used to replace old Pods by new ones | `Recreate` |
+| `timezone`                 | Timezone the Tautulli instance should run as, e.g. 'America/New_York' | `UTC` |
+| `puid`                     | process userID the Tautulli instance should run as | `1001` |
+| `pgid`                     | process groupID the Tautulli instance should run as | `1001` |
+| `Service.type`          | Kubernetes service type for the Tautulli GUI | `ClusterIP` |
+| `Service.port`          | Kubernetes port where the Tautulli GUI is exposed| `6789` |
```
Comment (billimek) [inline tautulli/README.md]: should this port be 8181 instead of 6789?

**Comment by onedr0p** [comment]
My copy pasta skills are off the mark, I'll fix these issues and the ones for Ombi later today. Thanks!

**Comment by onedr0p** [comment]
Updated

**Comment by billimek** [comment]
@onedr0p thank you so much for adding this chart!

**Pull request merged**